django_celery_results/models.py

```diff
@@ -63,6 +63,10 @@ class TaskResult(models.Model):
         verbose_name=_('Result Data'),
         help_text=_('The data returned by the task.  '
                     'Use content_encoding and content_type fields to read.'))
+    date_created = models.DateTimeField(
```

Comment:
I feel like it would be a lot safer to set null=True, otherwise it's too easy to throw an error during migration if something doesn't get a value set.

Reply:
Oh, I guess date_done doesn't allow nulls either.  Okay, this is probably fine then.